modules/simulate-network-policies-ng20.adoc

```diff
@@ -6,13 +6,17 @@
 = Simulating network policies from the network graph
 
 [role="_abstract"]
-Your current network policies might allow unneeded network communications.
-To simulate the effects of a new set of network policies, use the network policy simulator. For information about using the network policy simulator to generate policies, see "Generating network policies in the network graph."
+Your current network policies might allow unneeded network communications. You can also use the network policy generator to create network policies that restrict ingress traffic to the computed baselines for a set of deployments.
+
+[NOTE]
+====
+The Network Graph does not display the generated policies in the visualization. Generated policies are only for ingress traffic and policies are not generated that restrict egress traffic.
```

Comment:
Suggestion: not too sure but I feel like the sentence might be clearer with a little reordering

```suggestion
The Network Graph does not display the generated policies in the visualization. Generated policies are only for ingress traffic and policies that restrict egress traffic are not generated.
```